include/nbl/asset/ICPUGraphicsPipeline.h

```diff
@@ -93,7 +93,7 @@ class ICPUGraphicsPipeline final : public ICPUPipeline<IGraphicsPipeline<ICPUPip
 
 		inline int8_t stageToIndex(const ICPUShader::E_SHADER_STAGE stage) const override
 		{
-			const auto stageIx = hlsl::findLSB(stage);
+			const auto stageIx = hlsl::findLSB(static_cast<std::underlying_type_t<ICPUShader::E_SHADER_STAGE>>(stage));
```

Comment:
either partial spec for C++ only with `is_enum` constraint...

or use `core::bitflag` instead of `static_cast<std::underlying_type_t`

Reply:
done